fake-api/README.md

```diff
@@ -0,0 +1,75 @@
+# Banking Fake REST API
+Fake JSON API services for banking application
+
+## Resources
+```
+https://my-json-server.typicode.com/shangan23/banking-api/users
```

Comment:
this url should be changed to current repo right ?

Reply:
done